src/jdk.jdwp.agent/share/native/libdt_socket/socketTransport.c

```diff
@@ -396,17 +396,16 @@ parseAddress(const char *address, struct addrinfo **result) {
 }
 
 /*
- * Input is sockaddr just because all clients have it.
+ * Input is in_addr just because all clients have it.
  */
```

Comment:
The comment does not make sense anymore: in_addr represents IPv4 address, in6_addr represents IPv6 address.
Could you remove it please.